### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by ChristineStawitz-NOAA** [NOAA-FIMS/FIMS#423] [Announcements]
This is the first version of the Fisheries Integrated Modeling System (FIMS). At this stage, FIMS can be used to estimate a simple age-structured population model such as case 0 in Li et al. (2021)’s manuscript. Please refer to the [collaborative workflow](https://noaa-fims.github.io/collaborative_workflow/) for a description of the model functionality and completeness. Documentation can be found [here](https://noaa-fims.github.io/FIMS-docs/).

**Full Changelog**: https://github.com/NOAA-FIMS/FIMS/commits/v0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/NOAA-FIMS/FIMS/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by Craig44** [comment]
Congratulations on the first release!!

Is it possible to release the package with the vignettes already built with in the package so we can use `vignette(package = "FIMS")`. The first thing I wanted to do was run through an example, but if one exists, it wasn't obvious to me where I could see it. I know there is [fims-demo.Rmd](https://github.com/NOAA-FIMS/FIMS/blob/main/vignettes/fims-demo.Rmd) markdown file but Is this available as an html somewhere? (sorry if I have missed it).

Also a minor thing but the package [version](https://github.com/NOAA-FIMS/FIMS/blob/25a240574edefbc322ba64e7613e79d3b182f829/DESCRIPTION#L3C6-L3C17) doesn't match the release version which you may want to align or add some check on this in the future.

**Reply by ChristineStawitz-NOAA** [reply]
Thanks for checking! We are planning to make this a pkgdown site in the next month or so, at that point the vignette will be on the core website. Right now it is hosted here: https://connect.fisheries.noaa.gov/fims-demo/

I will fix the package version now!

**Comment by kellijohnson-NOAA** [comment]
I am closing this announcement as we are on version 0.9.3 and we haven't been using the Discussion board as a place to repost release notes. We will be thinking about how to announce releases in the immediate future. Stay tuned 😄.
